content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -1,225 +1,178 @@
-**Save**.
-
-Managing Product Variants
-=========================
-
-Introduction
-------------
-
-The examples below are all based on this product template that has two
-variant attributes :
+.. tip::
+   Similar creation processes are accessible through the Purchase, Inventory, and eCommerce
+   applications.
 
--  T-Shirt B&C
+Managing product exclusions
+---------------------------
 
-   -  Color: Red, Blue, White
+The following examples are all based on a product template that has two attributes:
 
-   -  Size: S, M, L, XL, XXL
+- T-Shirt
 
-Managing combination possibilities
-----------------------------------
+  - Color: Blue, Red, White
+  - Size: S, M, L, XL, XXL
 
-By default, with the above product template, you get 15 different
-products (3 colors, 5 sizes). If the XXL size only exists for red and
-blue t-shirts, you can deactivate the white product variant.
+With the above product template, there are 15 different t-shirt variants in three different colors
+and five different sizes. If the white t-shirts are not available in the XXL size, then that
+variant can be deactivated.
 
-To do this, click on the **Variants** button, select the XXL, White
-T-shirt. From the product form, uncheck the **Active** box of the T-shirt
-White, XXL.
+To deactivate a particular product variant, go to :menuselection:`Sales --> Products --> Products`
+and select the relevant product. Click on the :guilabel:`Configure Variants` button, select the
+line item for the relevant attribute value (in this example, the :guilabel:`White Color`
+attribute), and then click on :guilabel:`Edit`. In the :guilabel:`Exclude for` section, click
+:guilabel:`Add a line` and select any product(s) and/or specific attribute values that are
+incompatible.
 
-.. image:: variants/variant04.png
+.. image:: variants/attributes-exclusions.png
    :align: center
-
-.. tip::
-
-  That deactivating a product is different than having an inventory of 0.
+   :alt: Excluding attributes.
 
 Setting a price per variant
 ---------------------------
 
-You can add a cost over the main price for some of the variants of a
-product.
+Extra costs can be added to the standard price for specific product variants.
 
-Once you have activated the variant option, you can add variants to your
-products. To do so, go to the Sales module, open :menuselection:`Sales --> Products`
-and click on the product you want to modify. Click on the **Variant Prices**
-button to access the list of variant values.
+To do this, open :menuselection:`Sales --> Products --> Products`, and click on the relevant
+product. Click on :guilabel:`Configure Variants` to access the list of product variant values.
 
-.. image:: variants/variant05.png
+Click on a variant value, then click :guilabel:`Edit`. In the :guilabel:`Value Price Extra` field,
+type in the additional cost for this particular value. This amount is added to the standard price.
+
+.. image:: variants/value-price-extra.png
```

Comment:
I don't love that this screenshot is so zoomed in, because you can't tell what page/form it's on. @meng-odoo if possible, can you take a new screenshot that shows more of the Product Variant Value form.

Reply:
@mivu-odoo @samueljlieber Updated image attached
![value-price-extra](https://user-images.githubusercontent.com/101904966/187515709-b1e0d5cf-45d1-4869-9185-179fea9e7c08.png)